triggers/sg_jira_event_trigger.py

```diff
@@ -9,6 +9,16 @@
 import logging
 import requests
 import urlparse
+
+try:
+    # Allow users to define their sensible data in a .env file and
```

Comment:
I think you mean `sensitive` data, or is this some english subtlety I am not aware of?

Reply:
That's what I get for copy/pasting from the other file without really reading it :)